lib/xenops_server.ml

```diff
@@ -937,7 +937,7 @@ let rec atomics_of_operation = function
     let vgpus = VGPU_DB.vgpus id in
     let pcis = PCI_DB.pcis id |> pci_plug_order in
     let vusbs = VUSB_DB.vusbs id in
-    let pcis_other = List.filter (is_not_nvidia_sriov vgpus) pcis in
```

Comment:
is `is_not_nvidia_sriov` used anymore?

Reply:
`is_not_nvidia_sriov` is still used for `VM_restore_devices`.